### PR TITLE
✨ springAds: adding 3P amp-ad adapter "springAds" for SPRING

### PR DESCRIPTION
**Pull request by renebaudisch**
implementation for advertiser spring to deliver ads for AXELSPRINGER SE on their publishers domains and further improvements to e.g. amp-story-ads not supported by adserver distributor

### CODE REVIEW

File: ads/springAds.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import {createCustomEvent} from '../src/event-helper';
+import {loadScript} from '../3p/3p';
+import {parseJson} from '../src/json';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function springAds(global, data) {
+  context.master.availableSlots = context.master.availableSlots || {};
```
Comment (lannka) [inline ads/springAds.js]: could you check if the following method meets your needs for master frame selection:

https://github.com/ampproject/amphtml/blob/a04bd32fda335cc87cc6e5be29580b837d924f4f/3p/3p.js#L172-L186

Comment (renebaudisch) [reply ads/springAds.js]: Goal is to have the master frame as director/handler, that's why a reference to the other adframes is needed

**Comment by lannka** [comment]
@renebaudisch please fix the travis failures so that I can merge it

**Comment by lannka** [comment]
@renebaudisch still failed with a lint error

**Comment by renebaudisch** [comment]
Yeah, sorry, changed it, but didn't commit...

**Pull request merged**